src/modules/tr/process.cpp

```diff
@@ -31,26 +32,129 @@ Module::ExecutionResult TRModule::process(ModuleContext &moduleContext)
                          sourceNeutronSQ_->name());
         return ExecutionResult::Failed;
     }
-
+    // Get target gr module
     auto *grModule = sqModule->sourceGR();
     if (!sqModule)
     {
         Messenger::error("A source GR module (in the target SQ module) must be set.\n");
         return ExecutionResult::Failed;
     }
+
+    // Retrieve weights
+    const auto &weights = moduleData.value<NeutronWeights>("FullWeights", sourceNeutronSQ_->name());
+    // Retrieve GR and SQ
     const auto unweightedGR = moduleData.value<PartialSet>("UnweightedGR", grModule->name());
+    auto unweightedSQ = moduleData.value<PartialSet>("UnweightedSQ", sqModule->name());
+    auto referenceSQ = moduleData.value<Data1D>("ReferenceData", sourceNeutronSQ_->name());
+    // Make weightedGR Partial set
+    PartialSet weightedGR;
+    weightedGR.setUpPartials(unweightedSQ.atomTypeMix(), false);
 
     // Get effective atomic density of underlying g(r)
     const auto rho = grModule->effectiveDensity();
-
+    // Create weightedTR PartialSet in rtstart file
     auto [weightedTR, wGRstatus] = moduleContext.dissolve().processingModuleData().realiseIf<PartialSet>(
         "WeightedTR", name_, GenericItem::InRestartFileFlag);
     if (wGRstatus == GenericItem::ItemStatus::Created)
         weightedTR.setUpPartials(unweightedGR.atomTypeMix(), false);
 
-    // Retrieve weights
-    const auto &weights = moduleData.value<NeutronWeights>("FullWeights", sourceNeutronSQ_->name());
+    // Get Q-range and window function to use for transformation of F(Q) to G(r)
+    auto refftQMin = refQMin_.value_or(0.0);
+    auto refftQMax = refQMax_.value();
+    if (refWindowFunction_ == WindowFunction::Form::None)
+        Messenger::print("[SETUP {}] No window function will be applied in Fourier transform of S(Q) to g(r).", name_);
+    else
+        Messenger::print("[SETUP {}] Window function to be applied in Fourier transform of S(Q) is {}.", name_,
+                         WindowFunction::forms().keyword(refWindowFunction_));
+
+    // FT Reference data to ReresentativeTotalGR
+    Fourier::sineFT(referenceSQ, 1.0 / (2 * PI * PI * rho.value()), refftQMin, qDelta_, refftQMax, refWindowFunction_,
+                    refQBroadening_);
+
+    // Get Q-range and window function to use for transformation of F(Q) to G(r)
+    auto repftQMin = repQMin_.value_or(0.0);
+    auto repftQMax = repQMax_.value();
+    if (repWindowFunction_ == WindowFunction::Form::None)
+        Messenger::print("[SETUP {}] No window function will be applied in Fourier transform of S(Q) to g(r).", name_);
+    else
+        Messenger::print("[SETUP {}] Window function to be applied in Fourier transform of S(Q) is {}.", name_,
+                         WindowFunction::forms().keyword(repWindowFunction_));
```

Comment:
This output is identical to the stuff above, but operates on different data, so we should be explicit in the text as to what the window/FT is being applied to, e.g. "in Fourier transform of total F(Q) to G(r)" in the present case.
```suggestion
        Messenger::print("TR: No window function will be applied in Fourier transform of S(Q) to g(r).");
    else
        Messenger::print("TR: Window function to be applied in Fourier transform of S(Q) is {}.",
                         WindowFunction::forms().keyword(repWindowFunction_));
```